modules/local/format_taxresults.nf

```diff
@@ -23,8 +23,8 @@ process FORMAT_TAXRESULTS {
         path(fastafile)
 
         output:
-	path("ASV_tax.tsv")
-	path("ASV_tax_species.tsv"), emit: tsv
+    path("ASV_tax.tsv")
+    path("ASV_tax_species.tsv"), emit: tsv
```

Comment:
not good

Reply:
The above three should be fixed in c2930bbb7218e17e079750dea6c8040fd4a440d8.